Framework/PythonInterface/mantid/plots/__init__.py

```diff
@@ -611,12 +619,33 @@ def plot(self, *args, **kwargs):
 
             def _data_update(artists, workspace, new_kwargs=None):
                 # It's only possible to plot 1 line at a time from a workspace
+                try:
+                    if new_kwargs:
+                        x, y, _, __ = plotfunctions._plot_impl(self, workspace, args, new_kwargs)
```

Comment:
Does repeated use of `_` trigger a static analysis warning? If not it might be better to just use single underscores for throwaway variables (Python does allow this).

Reply:
OK I've replaced the double underscore with a single underscore